src/aiperf/plot/exporters/png/single_run.py

```
@@ -0,0 +1,129 @@
+
+"""
+Single-run PNG exporter for time series plots.
+
+Generates static PNG images for analyzing a single profiling run.
+"""
+
+from pathlib import Path
+
+import plotly.graph_objects as go
+
+import aiperf.plot.handlers.single_run_handlers  # noqa: F401
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)